src/m1_facilitator/handlers/DeclaredDepositIntentsHandler.ts

```diff
@@ -0,0 +1,174 @@
+
+// Copyright 2020 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import BigNumber from 'bignumber.js';
+import DepositIntent from '../models/DepositIntent';
+import Message, { MessageStatus, MessageType } from '../models/Message';
+import GatewayRepository from '../repositories/GatewayRepository';
+import DepositIntentRepository from '../repositories/DepositIntentRepository';
+import MessageRepository from '../repositories/MessageRepository';
+import Utils from '../../common/Utils';
+import Logger from '../../m0_facilitator/Logger';
+
+/**
+ * It handles updates from DeclaredDepositIntents entity.
+ */
+export default class DeclaredDepositIntentsHandler {
+  /** Instance of DepositIntentRepository. */
+  private depositIntentRepository: DepositIntentRepository;
```

Comment:
```suggestion
  private readonly depositIntentRepository: DepositIntentRepository;
```
Same for other variables

Reply:
Done